edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.IoTHub/EdgeAgentConnection.cs

```diff
@@ -368,6 +380,68 @@ Task UpdateDeploymentConfig(TwinCollection desiredProperties)
         async Task<bool> WaitForDeviceClientInitialization() =>
             await Task.WhenAny(this.initTask, Task.Delay(DeviceClientInitializationWaitTime)) == this.initTask;
 
+        private static bool CheckIfTwinPropertiesAreSigned(TwinCollection twinDesiredProperties)
+        {
+            return JObject.Parse(twinDesiredProperties.ToString())["integrity"] != null;
+        }
+
+        public static bool LogTwinSignatureEventsAndReportIfFailed(TwinCollection twinDesiredProperties)
```

Comment:
why public?

Reply:
Initially was thinking to reuse the same events for EA and EH and later realized it can't be. will change it.